File: CognizantGencElevate/ArrayLast.java
import java.util.Scanner;

class ArrayLast {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter N :");
        int N = sc.nextInt();
        int [] arr = new int[10];

        // getting input for array
        for(int i=0; i<N;i++){
            arr[i] = sc.nextInt();
        }

        // checking if arr[i] last digit is divisible by 10
        for(int i=0;i<N;i++){
            int rem = arr[i]%10;
            if(rem%10==0){
                System.out.println("YES");
            }
        }
        sc.close();
    }

}

File: CognizantGencElevate/ArrayLast.py
def ArrayLast(num):
    # getting i/p for the Array
    Arr = []
    for i in range(N):
        Arr.append(int(input("Enter the number ")))
    
    # checking if Arr[j] last digit  is divisible by 10 
    for j in Arr:
        rem = j%10
        if(rem%10==0):
            print("YES")


N = int(input("Enter N : "))
ArrayLast(N)